docs/templates/addon/theme_barceloneta/index.rst

```diff
@@ -0,0 +1,34 @@
+=================
+theme_barceloneta
+=================
+
+.. topic:: Description
+
+    Adding a barceloneta theme to an existing addon package.
+
+
+With this ``subtemplate``, you can add a Plone theme to a Plone package.
+
+First create a Plone Addon Package:
+
+.. code-block:: sh
```

Comment:
please use .. code-block:: shell

This is also written in our styleguide -> https://docs.plone.org/about/contributing/rst-styleguide.html :)